### ISSUE TITLE
10.1.5: Druid Guardian specs appear not to load correctly

### ISSUE THREAD

**Issue by dekimsey** [NumyAddon/TalentLoadoutManager#9]
When I select (or import) a talent spec for guardian onto TLM, it leaves me with 1 talent point remaining on the spec tree. This may be related to the changes in 10.1.5. This appears to be consistent regardless of the talent tree import. It doesn't appear to round-trip correctly.

Thank you!

**Comment by Numynum** [comment]
do you know which talent isn't being selected properly? Or better yet, can you give me a loadout string to import myself? 🙂

**Comment by dekimsey** [comment]
That's a good question! I should have done that leg-work before submitting, sorry about that cruddy bug report.

Using this [example](https://www.wowhead.com/talent-calc/druid/guardian/DAPAREFBCYJCFUUExlBBOVVERIhUVFQZQBUCAA). With the Blizzard import code:
```
BgGAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQiIJSJUSSgkiESShCAJJIIBSSSikIFohAAAAAAICIhQA
```

I select Import -> Give it a name and enter the code above. TLM doesn't auto-switch the drop-down on the right (this may be normal, I didn't ask it to apply it), and shows 1 talent point remaining and a number of misassigned talents.

Here's a screenshot, I've noted the talents that should be there and aren't vs the ones that are assigned. Thank you!

![image](https://github.com/NumyAddon/TalentLoadoutManager/assets/90741/3415747f-099d-439a-9151-61059c698e94)

**Comment by Numynum** [comment]
very helpful, thanks 🙂 
there seems to be an issue with the various Berserk talents, I'll look into it

**Comment by Numynum** [comment]
seems I was a bit too quick to judge 🙂 
there are 2 issues happening here as far as I can tell

1. blizzard has reset some of your talents, this makes loadouts work rather buggy, even without any addons loaded; your best bet is to manually switch to the `TalentLoadoutManager` loadout in the default UI, spend enough talents to make sure blizzard stops complaining, and then you can use this addon as usual again
2. there's a bug when exporting loadouts when there are multiple talents with the same spellID (as is the case with Berserk); this is a bug I have fixed in the next version, but this didn't actually have anything to do with your issue :)

TL;DR, fill out your talent trees on any blizzard loadout you have, then the game should stop throwing a tantrum whenever TLM tries to switch loadouts for you 🙂

**Comment by dekimsey** [comment]
I'll give that a try this evening, thank you!

**Comment by dekimsey** [comment]
I got it working thank you for the response! It was finicky to do though, I did end up having to disable TalentLoadoutManager, apply the misising point on the `TalentLoadoutManager` save and then re-enable the addon before I could switch talent saves. Very strange
